src/inc/quic_datapath.h

```diff
@@ -495,7 +495,7 @@ CxPlatSendDataAlloc(
 _IRQL_requires_max_(DISPATCH_LEVEL)
 void
 CxPlatSendDataFree(
-    _In_ CXPLAT_SEND_DATA* SendData
+    _In_ CXPLAT_SEND_DATA* SendContext
```

Comment:
Change these back to `SendData` please. Same for others.

Reply:
Then I'll be changing them in datapath_winuser and datapath_winkernel. This was one of the codeql complaints.

Reply:
CodeQL wasn't complaining about these in the past. Why now? But yes, I'd rather fix the Windows code (which has been on my TODO list for a while). The types all got renamed, but the variables only partially did.

Reply:
The linux code used to have the right names. Since this was a straight copy from windows, that gave it the wrong names.

Reply:
I'll fix the datapath ones in this PR, then do another PR to fix all the other usages outside of the datapath in a separate PR.